tools/Replay.hpp

```diff
@@ -213,7 +213,7 @@ class StressSocketHandler : public WebSocketHandler
         std::cerr << "Attempt connect to " << uri << " for trace " << _trace << "\n";
         getNextRecord();
         _start = std::chrono::steady_clock::now() + std::chrono::milliseconds(delayMs);
-        _nextPing = _start + std::chrono::milliseconds((long)(std::rand() * 1000.0) / RAND_MAX);
+        _nextPing = _start + std::chrono::milliseconds(Util::rng::getNext() % 1000);
```

Comment:
Just commenting here to note down my understanding. We used to have a 0..1 random number here (rand() / RAND_MAX), so this used to be some time between 0 and 1 seconds. The new way just gives us a random integer, hence the %1000.